trillian/migrillian/core/controller.go

```diff
@@ -32,6 +32,7 @@ import (
 
 	"github.com/google/trillian/merkle/logverifier"
 	_ "github.com/google/trillian/merkle/rfc6962" // Register hasher.
```

Comment:
It's likely that this line is no longer needed (if there is no code that tries to infer the hasher from the Trillian tree config).

Reply:
Ah, yes, good spot! Removed.